lib/log/target.toit

```diff
@@ -27,8 +27,13 @@ Standard log service used when the system log service cannot
 class StandardLogService_ implements LogService:
   buffer_/bytes.Buffer ::= bytes.Buffer.with-initial-size 64
 
-  log level/int message/string names/List? keys/List? values/List? -> none:
+  log level/int message/string names/List? keys/List? values/List? timestamp/bool -> none:
     buffer ::= buffer_
+
+    if timestamp:
+      buffer.write "$(Time.monotonic-us / 1000)"
```

Comment:
The digits should be padded. Space is probably enough, but 0 would work too.
Not sure what the correct digit-amount is.